pkg/version/version.go

```diff
@@ -71,6 +85,46 @@ func Version() (string, error) {
 	return releaseName, nil
 }
 
+// ReleaseArchitecture returns the release image cpu architecture version.
+func ReleaseArchitecture() (string, error) {
+	ri, okRI := os.LookupEnv("OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE")
+	if okRI {
+		logrus.Warnf("Found override for release image (%s). Release Image Architecture is unknown", ri)
+		return "", nil
+	}
+	if strings.HasPrefix(releaseArchitecturesPadded, releaseArchitecturesPrefix) {
+		logrus.Warn("Release Image Architecture not detected. Release Image Architecture is unknown")
+		return "", nil
```

Comment:
This will happen when an older `oc`, without multi support, is used, right? Maybe "unknown" is also better than "" in this case?

Reply:
Correct. I'm good with "unknown".